pages/community/mission.md

```diff
@@ -34,3 +30,15 @@ to target our activities and actions to serving these three aspects:
   to support the establishment and expansion of RSE positions and
   groups within the research ecosystem including the material for
   making business justifications for RSEs.
+
+## 4. Diversity, Equality, and Inclusion
+
+  We will actively promote, encourage, and improve diversity throughout the
+  broader US Research Software Engineer Community consistent with our full DEI
+  mission statement. We will ensure we provide an inclusive environment with
+  equitable treatment for all and we will prioritize a program of Diversity,
+  Equity, and Inclusion (DEI) activities for our organization, led by a
+  dedicated team of active community members.
+
+  Please see our full [DEI mission statement.]({{site.url}}/dei-mission/)
```

Comment:
Just thinking ahead - this URL for the new site will need to be under a dei namespace. Either something like `/working-group/dei/` or just give working groups a base url like `/dei/`. So this will work for now,but just forward thinking it will need to be made into a redirect.